parsely-amp-tracking.php

```diff
@@ -0,0 +1,25 @@
+add_action( 'pre_amp_render_post', 'parsely_amp_add_actions' );
+function parsely_add_amp_actions() {
+    add_filter( 'amp_post_template_analytics', 'parsely_amp_add_analytics' );
+}
+
+function parsely_amp_add_analytics( $analytics ) {
+    global $parsely;
```

Comment:
This line can be deleted since we won't need `parsely` as a global within the `Parsely` class